IFC4x3/Properties/w/WaterLeakage_0nA2qw75z08e_duxVyg0IJ/DocProperty.xml

```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Maybe a ratio measure?

Reply:
> Maybe a ratio measure?

They speak about classes so IfcLabel seems about right.

Reply:
"Degree of water leakage estimated from water flow information" not sure I understand your comment @larswik .

Reply:
Sorry, they say (classes) in column K (row 108 - WaterLeakage) in [Box-excel](https://app.box.com/file/1073785034660?s=jg3ys2w7t14se9wmpl6g82z51j28a7xw). I don't know if @j-weil could clarify with e.g. an example that could be used as a note or something?

Reply:
@j-weil: Are you referencing ANSI B16-104 Classes I, II to VI? In such case, could you also explain the classes (descriptions).

Reply:
Decision by domain: leave as IfcLabel.